VocaDbWeb/Views/Album/Details.cshtml

```diff
@@ -416,7 +416,7 @@
 							<h4 class="album-review-title">@Model.LatestReview.Title</h4>
 						}
 
-						<p>@HtmlHelpers.FormatMarkdown(Model.LatestReview.Text)</p>
+						<div>@HtmlHelpers.FormatMarkdown(Model.LatestReview.Text)</div>

```

Comment:
I noticed the hard way that <p> tags cannot be inside other <p> tags, and if they do, strange things will happen.